src/bindings.rs

```diff
@@ -56,6 +135,25 @@ fn js_date_constructor(context: *mut q::JSContext) -> q::JSValue {
     date_constructor
 }
 
+#[cfg(all(feature = "bignum", feature = "num"))]
+fn js_bigint_function(context: *mut q::JSContext) -> q::JSValue {
```

Comment:
```suggestion
fn js_bigint_constructor(context: *mut q::JSContext) -> q::JSValue {
```

Reply:
It is not an object constructor, because bigints are created by calling `BigInt(10)`, but not `new BigInt(10)`. So I thought that this name would highlight this difference.

Reply:
Function is a bit non-descriptive though.

Maybe `_create`?